mobile/src/bundles/common/components/slider/slider.tsx

```diff
@@ -0,0 +1,64 @@
+import CommunitySlider from '@react-native-community/slider';
+import React, { useState } from 'react';
+import { type StyleProp, type ViewStyle } from 'react-native';
+
+import { Pressable, Text, View } from '~/bundles/common/components/components';
+import { Color, TextCategory } from '~/bundles/common/enums/enums';
+import { convertInputToDate } from '~/bundles/common/helpers/helpers';
+import { useCallback, useWindowDimensions } from '~/bundles/common/hooks/hooks';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { styles } from './styles';
+
+type SliderProperties = {
+    style?: StyleProp<ViewStyle>;
+};
+
+const minSliderValue = 0;
+const maxSliderValue = 120;
+const valueWidth = 70;
```

Comment:
These values should be passed via props. Let's add default component props to SliderProperties

Reply:
fix